src/router/index.js

```diff
@@ -93,6 +96,14 @@ const routes = [
           back: dashboard
         }
       },
+      {
+        path: "/invitation",
```

Comment:
To be consistent with the general naming convention of the project I would recommend to use a plural form for this route: `/invitations`.